--- libzippp-libzippp-v7.1-1.10.1/CMakeLists.txt.orig	2024-01-23 16:25:17.000000000 +0100
+++ libzippp-libzippp-v7.1-1.10.1/CMakeLists.txt	2026-03-20 12:44:38.303672224 +0100
@@ -73,6 +73,7 @@
 
 if (BUILD_SHARED_LIBS)
   target_compile_definitions(libzippp PRIVATE LIBZIPPP_EXPORTS)
+  set_target_properties(libzippp PROPERTIES VERSION 0.0.0 SOVERSION 0)
 else()
   set_target_properties(libzippp PROPERTIES OUTPUT_NAME "libzippp_static")
 endif()
